script/Ciel/Day 13/13_01_01_CIEL13_3_2.txt

```diff
@@ -146,56 +154,64 @@ The Executor known as the 'Bow' appears before me.
 [sha:c146d7d1a8ae2b563b392af84bb50a3b31a1526e]{
 -- Page 355, Offset 34258.
 -- 鎌首をもたげる杭打ち機。
-The head of the pile bunker gets raised.
+// Assuming 鎌首 here is like 'raises its head like a snake'?
+// The industrial design doesn't have much in the way of gooseneck features that I can make out
+The pile bunker rears its ugly head.
 }
 [sha:4d7658f21c4dbf231ea5359e8296153cb6d16db6]{
 -- Page 355, Offset 34259.
 -- ……獲物を刺激しないという配慮か。
-...Perhaps in an attempt to refrain from agitating her prey―――
+I wonder if she acted out of consideration for her prey...
 }
 [sha:5da185bfe42d420299a7e6c88afaf5ee2bb128df]{
 -- Page 355, Offset 34260.
 -- ゆらりと、代行者の姿はこの視界から消失し、
-in one fell swoop, the Executor vanishes from my sight, and―――
+In one smooth motion, the Executor vanishes from my sight―――
 }
 [sha:a441accd3bc5800686174426084756dfed3d75fc]{
 -- Page 356, Offset 34261.
 -- 「―――さようなら。せめて、これで―――」
-"―――Farewell. At the very least―――"
+// Feels like I need the target here to make the gist of the JP clear, seems like a safe bet.
+"―――Farewell. At least with this, you―――"
 }
 [sha:35c7b34999402967e4ab5ec898d7eff18ec76225]{
 -- Page 356, Offset 34262.
 -- 小さな、聞き取れないほど小さな声で呟いて。
-With a mutter too faint to make out.
+―――Whispering to herself in a voice so faint I can hardly hear it.
 }
 [sha:cc98b0577e74e52fa0be3a70a8f7f7cb2e380665]{
 -- Page 356, Offset 34263.
 -- 彼女は切っ先を吸血鬼の心臓に押し当てて、弾劾の引き鉄を引いた。
-She pushes the tip against the vampire's heart before pulling the trigger of denunciation.
+Jamming the tip of her weapon against the vampire's heart,
+ she pulls the trigger that heralds its denunciation.
 }
 [sha:45f7702a61282a1e467d65a1bd709f560da38516]{
 -- Page 357, Offset 34264. Mods: @n.
 -- 教会の鐘を思わせる、体内で反響する炸裂音。
-Explosions reverberate within my body like church bells ringing.
+The sound of an explosion reverberates through my body like the peal of church bells.
 }
 [sha:42dca569379e8178b8c0149bafa2e9ceb56a6ec4]{
 -- Page 357, Offset 34265.
 -- 「ご―――ふ」
-"Ga―――ah."
+"%{i}Gugh%{/i}―――"
 }
 [sha:4d9b1312c8948108816f438263280ea0b9ef46e9]{
 -- Page 357, Offset 34266.
 -- 逆流した血液が口からあふれだす。
-The blood that I regurgitated pours out of my mouth.
+Blood spurts violently out of my mouth.
 }
 [sha:6fb71c964ae7c5797d55fb42d5fbd31039556204]{
 -- Page 357, Offset 34267.
 -- 脳髄の中心が炸裂するような閃光のあと。
-After a flash of light that feels like it caused the center of my brain to rupture.
+I see a flash of light, as if the core of my brain had gone critical.
 }
 [sha:f61d39b9bad73695a11e96a435abce8eaf363447]{
 -- Page 357, Offset 34268.
 -- 消えていく自分のなか、バラバラと散っていく、古い本のような魂を、見た。
 // not sure how the lore works for this one, but *my* soul might be wrong?
-From within my fading self, I catch a glimpse of my soul――it's reminiscent of a worn-out book whose tattered pages have been strewn about.
+// ~のあと yoinked down here
+As the glow fades,
+ and my self fades with it,
+ I catch sight of a soul;
+ unravelling and scattering to the wind like the pages of a worn out book.
```

Comment:
obviously book-soul is Roa. Might as well remove the initial comment.

Also, couldn't the first two clauses be merged?

>As I fade away together with the glow, I...